components/src/status_im/ui/components/react.cljs

```diff
@@ -46,7 +46,13 @@
 
 (def text-class (get-class "Text"))
 (def text-input-class (get-class "TextInput"))
-(def image (get-class "Image"))
+(def image-class (get-class "Image"))
+(defn image [{:keys [source] :as props}]
+  (when (or (not (map? source))
+            (not (contains? source :uri))
```

Comment:
Could this be factored into a 'valid-source?' function ?

Reply:
done